pygfx/materials/_mesh.py

```diff
@@ -89,6 +92,25 @@ def side(self, value):
             raise ValueError(f"Unexpected side: '{value}'")
         self._bump_rev()
 
+    @property
+    def wireframe(self):
+        """Render geometry as a wireframe with the given thickness.
+        Default is 0 (i.e. render as polygons).
+        """
+        return float(self.uniform_buffer.data["wireframe"])
```

Comment:
Or should we have one prop to turn it on/off and one to define the thickness?

Reply:
I do think a boolean flag is more natural. I doubt users will want to tweak the thickness very often, so offering that separately makes sense to me.

Reply:
done